README.md

```diff
@@ -33,6 +33,9 @@ push an initial commit to your fork:
 
    This step will take several minutes.
 
+   **Please note** that the `setup.sh` script will detect which OS you're on.
+   If it's macOS, the script will automatically set `backend: docker` inside `config.yaml`. This config is to start the cluster in containers as there is no KVM on macOS. However, Firekube on Linux allows to run either with `backend: ignite` (ignite VMs) or `backend: docker` (containers). So the `setup.sh` script won't touch the backend value inside `config.yaml` if you're on Linux. In case you started Firekube (in containers) on macOS then switched to try it on Linux, please make sure that the backend value set to `backend: ignite` so that you can start the cluster on ignite / firecracker VMs.
```

Comment:
Two minor nits:
- "allows to run" strikes me as an unusual construct; I think it would read better as "will run", "allows a user to run", or "supports running"
- "backend value set" => "backend value is set"